openapi/multi-yaml/index.yaml

```diff
@@ -79,223 +79,141 @@ tags:
+        auth-token="${webComponentToken}" 
+        checkout-id="${checkout.id}">
+      </justifi-checkout>
       ```
 
-      ##### Example Response: Checkout Completed
-      ```json
-      {
-        "id": "chc_4KIFTRC1hmdQx1X6jssYLk",
-        "type": "checkout_completion",
-        "page_info": null,
-        "data": {
-          "payment_mode": "ecom",
-          "payment_token": "pm_autoclonedxyz789",
-          "payment_status": "succeeded",
-          "payment_error_code": null,
-          "payment_error_description": null,
-          "checkout_id": "cho_123xyz",
-          "payment_id": "py_123xyz",
-          "payment_method_id": "pm_autoclonedxyz789",
-          "status": "succeeded",
-          "created_at": "2024-10-29T21:25:36.692Z",
-          "updated_at": "2024-10-29T21:25:36.692Z",
-          "payment_response": {
-            "id": "py_123xyz",
-            "type": "payment",
-            "data": {
-              "amount": 10000,
-              "currency": "usd",
-              "status": "succeeded",
-              "payment_method": {
-                "card": {
-                  "id": "pm_autoclonedxyz789",
-                  "acct_last_four": "3222",
-                  "brand": "mastercard",
-                  "name": "Tifa Lockhart",
-                  "token": "pm_autoclonedxyz789"
-                }
-              }
-            }
-          }
-        }
-      }
+      #### Via API
+      ```
+      // Create checkout
+      const checkout = await fetch('https://api.justifi.ai/v1/checkouts', {
+        method: 'POST',
+        headers: {
+          'Authorization': `Bearer ${token}`,
+          'Sub-Account': processingSubAccountId,  // Processing sub account
+          'Content-Type': 'application/json'
+        },
+        body: JSON.stringify({
+          "amount": 1799,
+          "description": "Example item",
+          "payment_method_group_id": "pmg_walletGroupId" // Group from wallet account
+        })
+      });
+
+      // Complete checkout with wallet payment method
+      const completion = await fetch(`https://api.justifi.ai/v1/checkouts/${checkoutId}/complete`, {
+        method: 'POST',
+        headers: {
+          'Authorization': `Bearer ${token}`,
+          'Sub-Account': processingSubAccountId,  // Processing sub account
+          'Content-Type': 'application/json'
+        },
+        body: JSON.stringify({
+          "payment_token": "pm_walletPaymentMethodToken"
+        })
+      });
       ```
 
-      Note that when completing the checkout with a wallet payment method token, the system automatically creates a cloned payment method for the sub-account as shown in the response's payment_method.token field. 
-      This same token will be retrieved for subsequent payment requests for the same Sub-Account passing the same wallet token
-
-      3. Optionally organize payment methods into groups using the Payment Method Groups API
+      ## Updating Wallet Payment Methods
```

Comment:
I'd add something like: 'You can update a payment method via payment methods API (https://docs.justifi.tech/api-spec#tag/Payment-Methods/operation/UpdatePaymentMethod)'